flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/runtime/stream/sql/FunctionITCase.java

```diff
@@ -244,27 +241,25 @@ public void testAlterFunctionNonExists() {
 
         try {
             tEnv().executeSql(alterUndefinedFunction);
-            fail();
+            fail("unknown failure");
```

Comment:
assertThatThrownBy would remove the need for this

Reply:
This can't be automated easily